src/ApolloHooksMutation.re

```diff
@@ -36,72 +30,98 @@ type controlledVariantResult('a) =
   | Error(error)
   | NoData;
 
-module Make = (Config: Config) => {
-  [@bs.module "graphql-tag"] external gql: ReasonApolloTypes.gql = "default";
-
-  type mutationResult = {. "data": option(Config.t)};
-
-  [@bs.deriving abstract]
-  type options = {
-    [@bs.optional]
-    variables: Js.Json.t,
-    [@bs.optional]
-    client: ApolloClient.generatedApolloClient,
-    [@bs.optional]
-    refetchQueries,
-    [@bs.optional]
-    awaitRefetchQueries: bool,
-    [@bs.optional]
-    update: (ApolloClient.generatedApolloClient, mutationResult) => unit,
-  };
+[@bs.module "graphql-tag"] external gql: ReasonApolloTypes.gql = "default";
+
+type mutationResult('a) = {. "data": 'a};
```

Comment:
This was `option(Config.t)` before, maybe it should be `option('a)` now, where `'a` will be the data type from `Config`?

Reply:
Do you mean that 'a should be option('a), whoops good spot then!